Kernel/Bus/USB/USBTransfer.cpp

```diff
@@ -42,9 +41,13 @@ void Transfer::set_setup_packet(USBRequestData const& request)
 
 ErrorOr<void> Transfer::write_buffer(u16 len, void* data)
 {
-    VERIFY(len <= m_dma_buffer.size());
-    m_transfer_data_size = len;
-    memcpy(buffer().as_ptr(), data, len);
+    (void)len;
+    (void)data;
+    /*
+        VERIFY(len <= m_dma_buffer.size());
+        m_transfer_data_size = len;
+        memcpy(buffer().as_ptr(), data, len);
+    */
```

Comment:
Commented out code in the first commit. Not sure if this is replaced later, but if so the wip part here should be removed.